TypeCobol.Test/TestUtils.cs

```diff
@@ -40,5 +46,89 @@ public static void compareLines(string testName, string result, string expectedR
+        {
+            string pwd = Directory.GetCurrentDirectory();
+            return Path.Combine(Directory.GetParent(pwd)?.FullName, _report);
+        }
+
+        public static void CreateRunReport(string localDirectoryFullName, string cobolFileName,
+            CompilationUnit compiler = null, CompilationStats stats = null)
+        {
+
+            // Display a performance report
+            StringBuilder report = new StringBuilder();
+            report.AppendLine("Program properties :");
+
+            report.AppendLine("- " + (compiler?.CobolTextLines.Count ?? stats?.Line) + " lines");
+            report.AppendLine("- " + (compiler?.CodeElementsDocumentSnapshot.CodeElements.Count() ?? stats?.TotalCodeElements) + " code elements");
+            if (compiler != null)
+            {
+                report.AppendLine("");
+                report.AppendLine("First compilation performance");
+                report.AppendLine("- " + compiler.PerfStatsForText.FirstCompilationTime + " ms : text update");
+                report.AppendLine("- " + compiler.PerfStatsForScanner.FirstCompilationTime + " ms : scanner");
+                report.AppendLine("- " + compiler.PerfStatsForPreprocessor.FirstCompilationTime + " ms : preprocessor");
+                report.AppendLine("- " + compiler.PerfStatsForCodeElementsParser.FirstCompilationTime +
+                                  " ms : code elements parser");
+                report.AppendLine("- " + compiler.PerfStatsForTemporarySemantic.FirstCompilationTime +
+                                  " ms : temporary semantic class parser");
+                report.AppendLine("- " + compiler.PerfStatsForProgramCrossCheck.FirstCompilationTime +
+                                  " ms : cross check class parser");
+            }
+
+            if (stats != null)
+            {
+
+                report.AppendLine("");
+                report.AppendLine(compiler != null
+                    ? "Incremental compilation performance (average time)"
+                    : "Full compilation performance (average time)");
+                report.AppendLine("- " + stats.AverageTextUpdateTime + " ms : text update");
+                report.AppendLine("- " + stats.AverageScannerTime + " ms : scanner");
+                report.AppendLine("- " + stats.AveragePreprocessorTime + " ms : preprocessor");
+                report.AppendLine("- " + stats.AverageCodeElementParserTime + " ms : code elements parser");
+                report.AppendLine("- " + stats.AverateTemporarySemanticsParserTime +
+                                  " ms : temporary semantic class parser");
+                report.AppendLine("- " + stats.AverageCrossCheckerParserTime + " ms : cross check class parser");
+            }
+
+            string reportFile = "Report_" + cobolFileName.Split('.')[0] + "_" +
+                                DateTime.Now.ToString().Replace("/", "_").Replace(":", "_").Replace(" ", "_") + "_" +
```

Comment:
@cobarzanc can you use a date formatter? Otherwise this will not work with every culture.